recipes/threddsclient/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "threddsclient" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d9d2e5292a73a9be47b3fc06d0fa306f7cc9f9774acc72c2fee1b89d80fb071b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can this be `noarch: python`?

Reply:
right. I have updated the recipe. Thanks.